src/main/java/com/wcc/platform/domain/cms/attributes/Event.java

```diff
@@ -0,0 +1,13 @@
+package com.wcc.platform.domain.cms.attributes;
+
+import java.util.List;
+
+/**
+ * Record for a single event
+ */
+public record Event(String topic, Category category, String startDate, String endDate, String title,
```

Comment:
```suggestion
public record Event(String topic, Category category, DateTime startDate, DateTime endDate, String title, List<ProgramType> topics, List<Image> images
```

Reply:
@dricazenck all changes done. Have a look when you have time